### DISCUSSION TITLE
Error: Request failed with status code 405

### DISCUSSION THREAD

**Discussion by imiloo** [gitalk/gitalk#442] [General]
第一次配置gitalk页面反馈这个报错

**Comment by booxood** [comment]
配置贴出来看下？

**Reply by sisyphus1212** [reply]
gitalk:
  clientID:
  clientSecret:
  repo:
  owner:
  admin: ['name']
  language: zh-CN
  labels: ['Gitalk']
  perPage: 10
  pagerDirection: last
  distractionFreeMode: false
  createIssueManually: true
  # 默认 proxy 可能会失效，解决方法请见下方链接
  # The default proxy may be invalid, refer to the links for solutions
  # https://github.com/gitalk/gitalk/issues/429
  # https://github.com/Zibri/cloudflare-cors-anywhere
  #proxy: https://cors-anywhere.azm.workers.dev/https://github.com/login/oauth/access_token

**Comment by cnrkuo** [comment]
@Surmiloo 
https://github.com/gitalk/gitalk/issues/437
